common/locales/en/gear.json

```diff
@@ -1,7 +1,10 @@
 {
   "set": "Set",
+  "equipmentType" : "Equipment Type",
+  "klazz": "Class",
```

Comment:
lol that's totally my bad, I was trying to be consistent and apparently combined the java "clazz" convention with the "klass" that was used here.